code/modules/antagonists/blob/structures/_blob.dm

```diff
@@ -342,14 +317,93 @@
 		icon_state = "blob_damaged"
 		name = "fragile blob"
 		desc = "A thin lattice of slightly twitching tendrils."
-		brute_resist = 0.5
+		brute_resist = BLOB_BRUTE_RESIST
 	else if (overmind)
 		icon_state = "blob"
 		name = "blob"
 		desc = "A thick wall of writhing tendrils."
-		brute_resist = 0.25
+		brute_resist = BLOB_BRUTE_RESIST * 0.5
 	else
 		icon_state = "blob"
 		name = "dead blob"
 		desc = "A thick wall of lifeless tendrils."
-		brute_resist = 0.25
+		brute_resist = BLOB_BRUTE_RESIST * 0.5
+
+/obj/structure/blob/special	// Generic type for nodes/factories/cores/resource
+	// Core and node vars: claiming, pulsing and expanding
+	/// The radius inside which (previously dead) blob tiles are 'claimed' again by the pulsing overmind. Very rarely used.
+	var/claim_range	= 0
+	/// The radius inside which blobs are pulsed by this overmind. Does stuff like expanding, making blob spores from factories, make resources from nodes etc.
+	var/pulse_range = 0
+	/// The radius up to which this special structure naturally grows normal blobs.
+	var/expand_range = 0
+
+	// Spore production vars: for core, factories, and nodes (with strains)
+	var/mob/living/simple_animal/hostile/blob/blobbernaut/naut = null
+	var/max_spores = 0 
+	var/list/spores	= list()
+	var/spore_delay = 0
+	var/spore_cooldown = BLOBMOB_SPORE_SPAWN_COOLDOWN
+
+	// Area reinforcement vars: used by cores and nodes, for strains to modify
+	/// Range this blob free upgrades to strong blobs at: for the core, and for strains
+	var/strong_reinforce_range = 0 
+	/// Range this blob free upgrades to reflector blobs at: for the core, and for strains
+	var/reflector_reinforce_range = 0
+
+/obj/structure/blob/special/proc/reinforce_area(delta_time)	// Used by cores and nodes to upgrade their surroundings
+	if(strong_reinforce_range)
+		for(var/obj/structure/blob/normal/B in range(strong_reinforce_range, src))
+			if(DT_PROB(BLOB_REINFORCE_CHANCE, delta_time))
+				B.change_to(/obj/structure/blob/shield/core, overmind)
+	if(reflector_reinforce_range)
+		for(var/obj/structure/blob/shield/B in range(reflector_reinforce_range, src))
+			if(DT_PROB(BLOB_REINFORCE_CHANCE, delta_time))
+				B.change_to(/obj/structure/blob/shield/reflective/core, overmind)
+
+/obj/structure/blob/special/proc/pulse_area(mob/camera/blob/pulsing_overmind, claim_range = 10, pulse_range = 3, expand_range = 2)
+	if(QDELETED(pulsing_overmind))
+		pulsing_overmind = overmind
```

Comment:
On which conditions would the `pulsing_overmind` be qdeleted, and can you guarantee that `overmind` will be valid then?

Reply:
I have no idea, this is how it was, I simply moved this proc down a typepath